feature/pairalert/src/main/java/dev/arkbuilders/rate/feature/pairalert/presentation/main/PairAlertConditionScreen.kt

```diff
@@ -185,12 +204,56 @@ fun PairAlertConditionScreen(navigator: DestinationsNavigator) {
             }
         }
     }
+
+    state.editGroupReorderSheetState?.let {
+        EditGroupReorderBottomSheet(
+            sheetState = editGroupReorderSheetState,
+            state = it,
+            onSwap = { f, t -> viewModel.onSwapGroups(f, t) },
```

Comment:
May we have proper names for the lambda parameters here

Reply:
Fixed